app/javascript/packages/document-capture/components/file-input.tsx

```diff
@@ -240,6 +240,13 @@ function FileInput(props: FileInputProps, ref: ForwardedRef<any>) {
     }
   }
 
+  /**
+    @return {boolean} Whether to show file upload hint
+  */
+  function shouldShowHint() {
```

Comment:
We could probably make this a simple variable (and colocate with `hintId` and friends above):

```ts
const showInnerHint = !value && !isValuePending && isMobile && !!bannerText;
```

Reply:
done!

Reply:
had to switch to `!` `isMobile`